src/render/extract.rs

```diff
@@ -96,10 +96,8 @@ impl ExtractedTilemapTexture {
                     it is being extracted as a texture!",
                 );
                 let texture_size: TilemapTextureSize = image.size().into();
-                let tile_count_x =
-                    (texture_size.x + tile_spacing.x) / (tile_size.x + tile_spacing.x);
-                let tile_count_y =
-                    (texture_size.y + tile_spacing.y) / (tile_size.y + tile_spacing.y);
+                let tile_count_x = ((texture_size.x) / (tile_size.x + tile_spacing.x)).floor();
```

Comment:
You need to floor here because otherwise you can get tile counts that are larger than what's available in the atlas.